website/static/css/jest.css

```diff
@@ -302,32 +302,37 @@ body
   }
 }
 
-// Temporary fix for not ideal Docusaurus 2 layout
-// May be removed when we close https://github.com/facebook/docusaurus/issues/4083
-.main-wrapper.blog-wrapper .container .row {
-  // Blog post list
-  .col.col--2:first-child {
-    --ifm-col-width: 22%;
-    div[class^='sidebar'] {
-      padding-right: 1rem;
-    }
-  }
+/* Temporary fix for not ideal Docusaurus 2 layout */
+/* May be removed when we close https://github.com/facebook/docusaurus/issues/4083 */
```

Comment:
@slorber this is closed - any idea if we can remove the overrides here now?

Reply:
It depends on whether the default Docusaurrus blog layout satisfy you or not: https://docusaurus.io/blog/2021/01/19/docusaurus-2020-recap

We improved but you may still want to keep custom CSS. Worth removing this and see if it looks good to you.